.pre-commit-config.yaml

```diff
@@ -24,5 +24,9 @@ repos:
         name: yapf
         entry: yapf
         language: system
+        exclude:
+            (?x)^(
+                aiida_common_workflows/workflows/relax/submission_template.py|
+            )$
```

Comment:
I think you can replace this with
```
exclude: *exclude_files
```
since you have defined the `exclude_files` variable above so you can reference that here, given that the content is the same